public/docs/ts/latest/tutorial/toh-pt2.jade

```diff
@@ -100,208 +97,213 @@ code-example(language="sh" class="code-shell").
 
-      Learn more about `ngIf`, `ngFor` and other structural directives in the
-      [Structural Directives](../guide/structural-directives.html) and
-      [Template Syntax](../guide/template-syntax.html#directives) chapters.
+      Read more about `ngIf`, `ngFor`, and other structural directives in the
+      [Structural Directives](../guide/structural-directives.html) page and the
+      [Built-in directives](../guide/template-syntax.html#directives) section of the
+      [Template Syntax](../guide/template-syntax.html) page.
+
 
   :marked
-    The browser refreshes and we see the list of heroes but not the selected hero detail.
-    The `ngIf` keeps it out of the DOM as long as the `selectedHero` is undefined.
-    When we click on a hero in the list, the selected hero displays in the hero details.
-    Everything is working as we expect.
+    When the browser refreshes, the list of heroes displays, but the selected hero detail does not.
+    The `ngIf` keeps the selected hero detail out of the DOM as long as the `selectedHero` is undefined.
+    When you click a hero in the list, the selected hero displays in the hero details.
+
+    ### Style the selection
 
-    ### Styling the selection
+    While the selected hero displays in the details area, it's difficult to quickly locate that hero in the list above.
+    To fix that, you can apply the `selected` CSS class to the appropriate `<li>` in the master list.
+    For example, when a user selects Magneta from the heroes list,
+    it will pop out with a subtle background color.
 
-    We see the selected hero in the details area below but we can’t quickly locate that hero in the list above.
-    We can fix that by applying the `selected` CSS class to the appropriate `<li>` in the master list.
-    For example, when we select Magneta from the heroes list,
-    we can make it pop out visually by giving it a subtle background color as shown here.
 
   figure.image-display
     img(src='/resources/images/devguide/toh/heroes-list-selected.png' alt="Selected hero")
   :marked
-    We’ll add a property binding on `class` for the `selected` class to the template. We'll set this to an expression that compares the current `selectedHero` to the `hero`.
+    In the template, you can add a property binding on `class` for the `selected` class.
+    You can set the class to an expression that compares the current `selectedHero` to the `hero`.
 
-    The key is the name of the CSS class (`selected`). The value is `true` if the two heroes match and `false` otherwise.
-    We’re saying “*apply the `selected` class if the heroes match, remove it if they don’t*”.
-  +makeExample('toh-2/ts-snippets/app.component.snippets.pt2.ts', 'class-selected-1', 'app.component.ts (setting the CSS class)')(format=".")
+    The key is the name of the CSS class (`selected`). The value is `true` if the two heroes match and `false` if they don't.
```

Comment:
This sentence was lost because there is no `:marked` in front of it.
But the sentence is also awful and should be deleted.